add the DATABASE_SSLMODE env var (#3581)
mozilla/pontoon: docs/admin/deployment.rst

   Maximum number of tasks a Celery worker process can execute before it’s
   replaced with a new one. Defaults to 20 tasks.

``DATABASE_SSLMODE``

>    Controls if the database needs a secure connection with the app.
> 
>    On `development`` envs it is always `false`. On `production` the default is `true`.

Reviewer: ```suggestion
   Optional. Controls if the database needs a secure connection with the app. Default value is `False`.
```
